### DISCUSSION TITLE
Add Twitch Emotes to the BTTV global database to apply all effects to them (w! s! c! h! v! z! l! r! p!)

### DISCUSSION THREAD

**Discussion by Pafon22** [night/betterttv#6836] [Ideas]
Incorporate Twitch emotes into BTTV itself to be able to use them with all BTTV effects. 
As it currently stands, each modifier appears with its own symbol without applying the effect to the Twitch emote.
![image](https://github.com/night/betterttv/assets/22238807/0e119a6b-db12-475a-9b19-1d70dab0f081)

**Comment by night** [comment]
Closing as a duplicate of #5909, which covers applying BTTV emote modifiers to non-BTTV emotes (Twitch, FFZ, and 7TV). Please add your upvote and any extra detail over there so feedback stays consolidated.
